releases/preprod/app.yml

```diff
@@ -7,9 +7,9 @@ metadata:
   annotations:
     fluxcd.io/automated: "true"
     filter.fluxcd.io/appealReplyServiceApi: semver:~1.1.0
-    filter.fluxcd.io/appealsServiceApi: semver:~1.3.0
-    filter.fluxcd.io/documentServiceApi: semver:~1.0.0
-    filter.fluxcd.io/formsWebApp: semver:~1.25.2
+    filter.fluxcd.io/appealsServiceApi: semver:~1.7.0
+    filter.fluxcd.io/documentServiceApi: semver:~1.1.0
+    filter.fluxcd.io/formsWebApp: semver:~1.26.0
     filter.fluxcd.io/lpaQuestionnaireWebApp: semver:~1.5.0
```

Comment:
Do you also want:

```
filter.fluxcd.io/pdfServiceApi: semver:^1.0.0-0
```

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/releases/dev/app.yml#L15

Reply:
And if so:

```
    pdfServiceApi:
      replicaCount: 2
      image:
        repository: pinscommonukscontainers3887default.azurecr.io/pdf-service-api
        tag: 1.0.0

    gotenberg:
      replicaCount: 2
```

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/releases/dev/app.yml#L57-L64

Reply:
:)

yup. just come back here from forensically calculating the need to do ^